sktime/forecasting/ardl.py

```diff
@@ -216,8 +216,7 @@ class ARDL(_StatsModelsAdapter):
         "capability:pred_int": False,  # does forecaster implement proba forecasts?
         # CI and test flags
         # -----------------
-        "tests:skip_by_name": ["test_predict_time_index_with_X"],
-        # known failure in case of non-contiguous X, see issue #8787
```

Comment:
just a minor nitpick regarding documentation - remove the "CI and test flags" section, since this is now a capability tag.